src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleStitchingService.java

```diff
@@ -71,7 +71,7 @@ public JsonNode updateCase(CcdCallbackDto ccdCallbackDto) {
         return bundle;
     }
 
-    private CcdValue<CcdBundleDTO> bundleJsonToBundleDto(JsonNode jsonNode) throws IOException {
+    private CcdValue<CcdBundleDTO> bundleJsonToCcdValue(JsonNode jsonNode) throws IOException {
```

Comment:
refactor - method name was a lie

Reply:
hmmm sort of - it's converting it to a CcdValue of CcdBundleDTO. Saying it converts to CcdValue isn't the full story and it's less useful information (a CcdValue might contain anything)

Reply:
Fair enough, I can appreciate that. What about changing the method to actually return a BundleDto? Or shall I just revert this change?

Reply:
I think so